docs/academy/exercise1.md

```diff
@@ -47,7 +49,10 @@ Start by creating a new workspace, installing the dependencies, and cloning all
         ```
 
     === "Windows"
-        Download and run the [installer](https://developer.arm.com/-/media/Files/downloads/gnu-rm/10-2020q4/gcc-arm-none-eabi-10-2020-q4-major-win32.exec) to install `arm-none-eabi-gcc` and `arm-none-eabi-gdb`.
+        !!! note
+            For Windows systems you will also require to install the [Python](https://www.python.org/downloads/windows/), [MinGW](https://sourceforge.net/projects/mingw/) and [CMake](https://cmake.org/download/) programs.
```

Comment:
Why `Mac` has only `brew install armmbed/formulae/arm-none-eabi-gcc` as a dependency? No Python or CMake are needed? It's still so inconsistent.

Reply:
Also, why won't you show installation steps for windows, but you do for e.g. Linux?

Reply:
for linux and mac the installation is writing the terminal commands. For windows is downloading the files from links